src/components/Common/PageTitle.tsx

```diff
@@ -56,17 +36,6 @@ export default function PageTitle({
       ref={divRef}
       className={classNames(!isInsidePage && "mb-2 md:mb-4", className)}
```

Comment:
might as well change the deprecated usage of `className` to `cn` along with this PR in this file.